packages/cli/src/outputting.js

```diff
@@ -38,9 +38,14 @@ export function globalOperation(msg) {
 /**
  * @param {string} msg
  * @param {import('@bilt/types').PackageInfo} packageInfo
+ * @param {number | null} current
```

Comment:
I usually prefer not putting `null` in my code. `undefined` is the only one I use.

Reply:
ok